soc/arm/nordic_nrf/nrf53/soc.c

```diff
@@ -415,6 +415,10 @@ static void rtc_pretick_rtc_isr_hook(void)
 void rtc_pretick_rtc0_isr_hook(void)
 {
 	rtc_pretick_rtc_isr_hook();
+
+	if (IS_ENABLED(CONFIG_SOC_NRF53_RTC_PRETICK)) {
```

Comment:
This `if` is redundant as the whole thing is inside https://github.com/zephyrproject-rtos/zephyr/blob/bf4652e842c56737b66fe4d50b4dcef41010a3bb/soc/arm/nordic_nrf/nrf53/soc.c#L377

And why not do this event clearing in `rtc_pretick_rtc_isr_hook` (it is common for RTC0 and RTC1)?

Reply:
fixed